tests/software/nanoplot/test.yml

```diff
@@ -0,0 +1,4 @@
+- name: nanoplot
+  command: nextflow run ./tests/software/nanoplot -entry test_nanoplot -c tests/config/nextflow.config
+  tags:
+    - nanoplot
```

Comment:
Need to add some tests here, ideally md5sums but failing that we can use the `contains` statement to see if the file contains a particular string and failing that we can just check for file existence. See [here](https://pytest-workflow.readthedocs.io/en/stable/#test-options)

Reply:
Thanks for the suggestion! I have added a check `contains: - "report"` for the `NanoPlot_report.html` output